cpp/include/cuvs/neighbors/cagra.h

```diff
@@ -416,6 +419,32 @@ cuvsError_t cuvsCagraSerialize(cuvsResources_t res,
                                cuvsCagraIndex_t index,
                                bool include_dataset);
 
+/**
+ * Save the CAGRA index to file in hnswlib format.
+ *
+ * Experimental, both the API and the serialization format are subject to change.
```

Comment:
Please describe the assumptions wrt cuVS and hnswlib so that it's blatantly clear for users. For example, this requires the user to use cuVS as a proxy for hnswlib because we've patched hnswlib to make this possible, even though we're writing to the hnswlib format.